tools/clang/test/HLSLFileCheck/hlsl/objects/Texture/sampleInteger.hlsl

```diff
@@ -42,10 +32,12 @@ float4 main(float2 coord : TEXCOORD0) : SV_Target {
   res += g_tex0.SampleBias(g_samp, coord, 0.0);
   res += g_tex0.SampleLevel(g_samp, coord, 0.0);
   res += g_tex0.SampleGrad(g_samp, coord, 0.0, 0.0);
+#ifndef PASS
   res += g_tex0.SampleCmp(g_sampCmp, coord, 0.0);
   res += g_tex0.SampleCmpLevelZero(g_sampCmp, coord, 0.0);
 #if  __SHADER_TARGET_MAJOR > 6 || (__SHADER_TARGET_MAJOR == 6 && __SHADER_TARGET_MINOR >= 7)
```

Comment:
I don't think the line below is ever executed. All tests with SM=6.7 have -D Pass defined, and so this whole block will be skipped. The run lines without PASS defined are all shader model 6.6, so this if condition would never be true. 
Also, I wonder if there's a better more descriptive term? For example, rather than PASS, how about "SKIP_INT_SAMPLE_COMPARES"
Then it can be `#ifdef SKIP_INT_SAMPLE_COMPARES` rather than `#ifndef PASS`

Reply:
Fixed.